clang/include/clang/Basic/Features.def

```diff
@@ -99,6 +99,9 @@ FEATURE(memory_sanitizer,
 FEATURE(thread_sanitizer, LangOpts.Sanitize.has(SanitizerKind::Thread))
 FEATURE(dataflow_sanitizer, LangOpts.Sanitize.has(SanitizerKind::DataFlow))
 FEATURE(scudo, LangOpts.Sanitize.hasOneOf(SanitizerKind::Scudo))
+FEATURE(swiftcc,
```

Comment:
This is not a `FEATURE`, it's an `EXTENSION`: https://github.com/llvm/llvm-project/blob/33374c445d31bfe5b8c638900a09e85d5bc409ee/clang/include/clang/Basic/Features.def#L20

I think the other swift calling convention should be fixed at the same time, that looks like a simple oversight.